pkg/resource/deploy/step.go

```diff
@@ -1505,6 +1505,8 @@ func Color(op display.StepOp) string {
 		return colors.SpecUpdate
 	case OpReadDiscard, OpDiscardReplaced:
 		return colors.SpecDelete
+	case OpRemovePendingReplace:
+		return colors.SpecUnimportant
```

Comment:
Again, this feels like the right choice to me -- it's unimportant, an implementation detail (?) -- but happy to adapt if wrong.